api/reference/providers.adoc

```diff
@@ -452,3 +453,247 @@ NOTE: When managing providers of _provider_ provider_class, Tagging and Policy m
 
+[[provider-examples]]
+=== Provider Examples
+
+Below you will find example requests for creating various providers.
+
+Ansible Tower Automation Manager:
+[source,json]
+----
+{
+  "type": "ManageIQ::Providers::AnsibleTower::AutomationManager",
+  "name": "API test",
+  "api_version": "3.1.2",
+  "provider_id": 10000000000012,
+  "credentials": {
+    "userid": "userid",
+    "password": "password"
+  }
+}
+----
+
+AWS:
+[source,json]
+----
+{
+  "type": "ManageIQ::Providers::Amazon::CloudManager",
+  "provider_region": "us-east-1",
+  "name": "Amazon Example",
+  "credentials": {
+    "userid": "<access key id>",
+    "password": "<secret access key>"
+  }
+}
+----
+
+Azure:
+[source,json]
+----
+{
+  "type": "ManageIQ::Providers::Azure::CloudManager",
+  "name": "Azure Example",
+  "provider_region": "eastus",
+  "uid_ems": "<tenant_id>",
+  "subscription": "<subscription_id>",
+  "credentials": {
+    "userid": "<client id>",
+    "password": "<client key>"
+  }
+}
+----
+
+Google Compute Engine:
+[source,json]
+----
+{
+   "name":"GCE Example",
+   "type":"ManageIQ::Providers::Google::CloudManager",
+   "provider_region":"us-central1",
+   "project":"GCE Project Name",
+   "connection_configurations":[
+      {
+         "endpoint":{
+            "role":"default"
+         },
+         "authentication":{
+            "authtype":"default",
+            "type":"AuthToken",
+            "auth_key":"<auth_key_here>"
+         }
+      }
+   ]
+}
+----
+
+Kubernetes Container Manager:
+[source,json]
+----
+{
+  "type": "ManageIQ::Providers::Kubernetes::ContainerManager",
+  "name": "Kubernetes Test",
+  "connection_configurations": [
+    {
+      "endpoint": {
+        "hostname": "00.0.00.00",
+        "port": "6443",
+        "security_protocol": "ssl"
+      },
+      "authentication": {
+        "authtype": "bearer",
+        "auth_key": "<token_here>"
+      }
+    }
+  ]
+}
+----
+
+Microsoft System Center VMM:
+[source,json]
+----
+{
+  "type": "ManageIQ::Providers::Microsoft::InfraManager",
+  "name": "SCVMM API example",
+  "api_version": "2.1.0",
+  "connection_configurations": [
+    {
+      "endpoint": {
+        "hostname": "00.0.00.00",
+        "port": "443",
+        "security_protocol": "ssl"
+      },
+      "authentication": {
+        "username": "username",
+        "password": "password"
+      }
+    }
+  ]
+}
+----
+
+Openshift Container Manager:
+[source,json]
+----
+{
+   "name":"ocp-example",
+   "port":8443,
+   "type":"ManageIQ::Providers::Openshift::ContainerManager",
+   "hostname":"ocp.example.com",
+   "connection_configurations":[
+      {
+         "endpoint":{
+            "role":"default",
+            "security_protocol":"ssl-without-validation",
+            "verify_ssl":0
+         },
+         "authentication":{
+            "authtype":"bearer",
+            "type":"AuthToken",
+            "auth_key":"<auth key>"
+         }
+      }
+   ]
+}
+----
+
+Openstack Cloud Manager:
```

Comment:
Is it possible an Openstack Cloud Manager also link to an existing Openstack Infra Provider?

Reply:
One more nitpick - please capitalize the S in OpenStack.